Cannot reflect unexported struct

**Describe the bug**
Cannot reflect unexported struct

**To Reproduce**
Create an unexported struct (`type myStruct struct`) and save with ReflectFrom

**Expected behavior**
Be able to save unexported structs from the same package it's defined in

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Additional context**
I'm a noob to golang, but it seems the `reflect` library can work with unexported structs. I am reflecting a config struct used by the same package that defines it (with getters/setters external access), but getting a reflection error:
```
2020/02/16 06:04:39 created empty ini
2020/02/16 06:04:39 reflecting struct
panic: reflect.Value.Interface: cannot return value obtained from unexported field or method

goroutine 1 [running]:
reflect.valueInterface(0x561502d075a0, 0xc0000866e8, 0x1b8, 0x1, 0x561502cf0000, 0xf)
	/usr/lib/go/src/reflect/value.go:1014 +0x1be
reflect.Value.Interface(...)
	/usr/lib/go/src/reflect/value.go:1003
gopkg.in/ini%2ev1.(*Section).reflectFrom(0xc000086770, 0x561502d69d40, 0xc000086690, 0x199, 0xc000086690, 0x199)
	/root/go/src/gopkg.in/ini.v1/struct.go:539 +0x258
gopkg.in/ini%2ev1.(*Section).ReflectFrom(0xc000086770, 0x561502d5a8a0, 0xc000086690, 0xc000086770, 0x0)
	/root/go/src/gopkg.in/ini.v1/struct.go:598 +0x14a
gopkg.in/ini%2ev1.(*File).ReflectFrom(0xc0000e20c0, 0x561502d5a8a0, 0xc000086690, 0xc0000ddb40, 0xc000096000)
	/root/go/src/gopkg.in/ini.v1/struct.go:603 +0x57
gopkg.in/ini%2ev1.ReflectFromWithMapper(...)
	/root/go/src/gopkg.in/ini.v1/struct.go:609
gopkg.in/ini%2ev1.ReflectFrom(...)
	/root/go/src/gopkg.in/ini.v1/struct.go:614
```

--- a/struct.go
+++ b/struct.go
@@ -564,6 +564,10 @@ func (s *Section) reflectFrom(val reflect.Value) error {
 	typ := val.Type()
 
 	for i := 0; i < typ.NumField(); i++ {
+		if !val.Field(i).CanInterface() {
+			continue
+		}
+
 		field := val.Field(i)
 		tpField := typ.Field(i)
 

--- a/struct_test.go
+++ b/struct_test.go
@@ -260,7 +260,6 @@ func Test_MapToStruct(t *testing.T) {
 			So(ts.TimePtrNil, ShouldEqual, nil)
 			So(*ts.DurationPtr, ShouldEqual, 0)
 			So(ts.DurationPtrNil, ShouldEqual, nil)
-
 		})
 
 		Convey("Map section to struct", func() {
@@ -440,12 +439,13 @@ func Test_ReflectFromStruct(t *testing.T) {
 			GPA       float64
 			Date      time.Time
 			NeverMind string `ini:"-"`
+			ignored   string
 			*Embeded  `ini:"infos" comment:"Embeded section"`
 		}
 
 		t, err := time.Parse(time.RFC3339, "1993-10-07T20:17:05Z")
 		So(err, ShouldBeNil)
-		a := &Author{"Unknwon", true, nil, 21, 100, 2.8, t, "",
+		a := &Author{"Unknwon", true, nil, 21, 100, 2.8, t, "", "ignored",
 			&Embeded{
 				[]time.Time{t, t},
 				[]string{"HangZhou", "Boston"},

